nck/utils/stdout_to_log.py

```diff
@@ -0,0 +1,68 @@
+import logging
+import sys
+import httplib2
+
+
+class STDoutToLog:
+    def __init__(self, logger_name, level):
+        self.content = []
+        self.logg = logging.getLogger(logger_name)
+        self.level = level
+
+    def write(self, string):
+        if not string.endswith("\n"):
+            self.content.append(string)
+        else:
+            debug_info = (
+                "".join(self.content)
+                .replace("\\r", "")
+                .encode("latin1")
+                .decode("unicode-escape")
+                .encode("latin1")
+                .decode("utf-8")
+                .replace("'", "")
+            )
+
+            debug_info = "\n".join([ll.rstrip() for ll in debug_info.splitlines() if ll.strip()])
+            self.logg.log(self.level, debug_info)
+            self.content = []
+
+    def flush(self):
+        pass
+
+
+def http_log(logger_name, level=logging.DEBUG):
+    def decorator(func):
+        def wrapper(*args, **kwargs):
+            httplib2.debuglevel = 4
```

Comment:
Is there a constant with an explicit name that hides this magic number 4. Would be great for readability.

Reply:
No there isn't. But after further checking it seems that any value greater than 0 activate debug mode.
```suggestion
            httplib2.debuglevel = 1
```
Do you prefer it like that?

Reply:
In the [documentation](https://httplib2.readthedocs.io/en/latest/libhttplib2.html#httplib2.debuglevel), they say that this is the amount of information we want to print. So, with 1, we may miss some info. Let's keep it that way 🙂